features/admin/step_definitions/drop_enrollment_members_steps.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+Given(/drop_enrollment_members feature is enabled/) do
+  EnrollRegistry[:drop_enrollment_members].feature.stub(:is_enabled).and_return(true)
+end
+
+Given(/^User with multiple member enrollment exists$/) do
+  create_multiple_member_enrollment_for_family_with_one_minor
+end
+
+When(/^Hbx Admin clicks on the Drop Enrollment Members button$/) do
+  find_link(l10n('admin_actions.drop_enrollment_members')).click
```

Comment:
Due to the nature of our _dropdown.html.erb adding a data-cuke attribute or a css class would require extra logic to be added in the view. Because this logic would only be relevant for testing I've decided to avoid it for this step specifically.